examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/SW4STM32/STM32F401RE-Nucleo/xively_demo_stm32f4xx-nucleo-wifi/.cproject

```diff
@@ -5,7 +5,7 @@
 			<storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539" moduleId="org.eclipse.cdt.core.settings" name="Debug">
 				<macros>
 					<stringMacro name="XI_STMCUBE_PATH" type="VALUE_PATH_DIR" value="/Users/atigyi/Downloads/STM32CubeExpansion_WIFI1_V2.1.1"/>
```

Comment:
My main concern is that in the tutorial we are telling the users to set the same variable to 2 different folders, depending on which project they're at. It saves us a couple lines of explaining, but I believe it may confuse the user.

Anyway, the tutorial explains how to set everything up, so I'm not particularly worried about the variable name.